src/catalog/catalog_entry/duck_constraint_entry.cpp

```diff
@@ -0,0 +1,26 @@
+#include "duckdb/catalog/catalog_entry/duck_constraint_entry.hpp"
+#include "duckdb/storage/data_table.hpp"
+
+namespace duckdb {
+
+DuckConstraintEntry::DuckConstraintEntry(Catalog &catalog, SchemaCatalogEntry &schema, CreateConstraintInfo &info)
+    : ConstraintCatalogEntry(catalog, schema, info) {
+}
+
+DuckConstraintEntry::~DuckConstraintEntry() {
+}
+
+string DuckConstraintEntry::GetSchemaName() const {
+	return "";
```

Comment:
I am adding constraints as currently indexes are added. 

Indexes have this `shared_ptr<DataTableInfo> info` variable, which is contains schema name and table name like so: `info->schema`, `info->table`. 

How should I do the same for constraints?